floater/test/test_generators.py

```diff
@@ -113,5 +113,7 @@ def test_to_mitgcm_format(fs, tmpdir):
 	    assert os.path.exists(filename)
             array = np.fromfile(filename, dtype='>f4')
             assert len(array)==(fs.Nx*fs.Ny + 1 )* 9
+            npart = fs.N
```

Comment:
And they do...but we want them to fail! So we can find the bug.

Stand by for an hour. I am going to make another quick PR to try this differently.